ansible/roles_ocp_workloads/ocp4_workload_dm7_foundations_elt/templates/clusterresourcequota.j2

```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: quota.openshift.io/v1
+kind: ClusterResourceQuota
```

Comment:
Is this workload intended to run on a shared cluster? If so, we already have a clusterresourcequota for users when they order access. How is this working with that?